examples/counter/store/switchReducer.js

```diff
@@ -0,0 +1,19 @@
+// Note: this is meant to be provided in a separate package.
+// I'm only including it here for discussion.
+
+const SWITCH_REDUCER = { type: 'redux-switch/SWITCH_REDUCER' };
```

Comment:
Ah, forgot about it. Yeah, it's probably better to leave it in because I can't imagine a siutation where you'd want to handle it.